packages/cozy-harvest-lib/src/locales/en.json

```diff
@@ -155,7 +155,7 @@
       },
       "UNKNOWN_ERROR": {
         "title": "Connection error",
-        "description": "An unknown error has occurred. You can try to update your data. If the problem persists, please contact us at [claude@cozycloud.cc](mailto:claude@cozycloud.cc)."
+        "description": "An unknown error has occurred. You can try to update your data. If the problem persists, please contact us at [%{supportMail}](mailto:%{supportMail})."
```

Comment:
just before this sentence we've another mail adresse https://github.com/cozy/cozy-libs/blob/ec14a4ebae901767536e9c3ee7522c995534f1db/packages/cozy-harvest-lib/src/locales/en.json#L154 

we should also fix it